public/api-merged.json

```diff
@@ -14353,6 +14488,10 @@
           "type": "string",
           "x-go-name": "Group"
         },
+        "hidden": {
+          "type": "boolean",
+          "x-go-name": "Hidden"
+        },
```

Comment:
these changes are unrelated to ours, but seem to be included when running `make swagger-api-spec-mac`

Reply:
This change in the model was introduced [here](https://github.com/grafana/grafana-enterprise/pull/2989/files#diff-4a6a200bd703d6cdc5258455569746f486c44c861ced31ea423abc18b41cb3ba) but the specification was not updated then. It's fine to keep it.